nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/FetchFileTransfer.java

```diff
@@ -254,36 +254,38 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             transfer = transferWrapper.getFileTransfer();
         }
 
+        Relationship failureRelationship = null;
+        boolean closeConnOnFailure = false;
+        boolean provenanceEventOnFailure = false;
```

Comment:
Reviewing the changes, it looks like this conditional check can be removed. Although the previous failure handling block did not call the Provenance Reporter on communications failure, this should be changed to always call `getProvenanceReporter().route()` for failures.